frontend/ui/elements/physical_buttons.lua

```diff
@@ -19,6 +19,28 @@ local PhysicalButtons = {
     },
 }
 
+if Device:hasScreenKB() or Device:hasSymKey() then
```

Comment:
```suggestion
if Device: useDPadAsActionKeys() then
```

Reply:
Please make it hasDPad and useDPadAsActionKeys in that case.